Example/PrebidDemo/PrebidDemoObjectiveC/Info.plist

```diff
@@ -41,14 +41,18 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
-	<key>GADIsAdManagerApp</key>
-	<true/>
 	<key>UISupportedInterfaceOrientations~ipad</key>
 	<array>
 		<string>UIInterfaceOrientationPortrait</string>
 		<string>UIInterfaceOrientationPortraitUpsideDown</string>
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+
+    <key>GADIsAdManagerApp</key>
+    <true/>
+    <key>gad_preferred_webview</key>
```

Comment:
Do we need to set this even if UIWebView is removed completely from the SDK?

Reply:
This case is useful when AM SDK has a version that is lower then `v7.55.0` where `UIWebView` is used by default